src/slider/stories/MultiSlider.component.tsx

```diff
@@ -0,0 +1,159 @@
+import * as React from "react";
+import { VisuallyHidden } from "reakit";
+
+import {
+  useSliderState,
+  SliderTrack,
+  SliderThumb,
+  SliderInput,
+} from "renderless-components";
+
+interface AppProps {
+  /**
+   * Label for the slider
+   *
+   * @default Styled
+   */
+  label?: string;
+  /**
+   * True, if thumb needs a tip to show it's current percent
+   */
+  showTip?: boolean;
+  /**
+   * The `value` of the slider indicator.
+   *
+   * If `undefined`/`not valid` the slider bar will be the optimum of min & max
+   * @default [25,50,75]
+   */
+  values?: number[];
+  /**
+   * The minimum value of the slider
+   * @default 0
+   */
+  min?: number;
+  /**
+   * The maximum value of the slider
+   * @default 100
+   */
+  max?: number;
+  /**
+   * The step in which increments/decrements have to be made
+   * @default 1
+   */
+  step?: number;
+  /**
+   * If `true`, the slider will be disabled
+   * @default false
+   */
+  isDisabled?: boolean;
+  /**
+   * Orientation of the slider
+   * @default "horizontal"
+   */
+  orientation?: "horizontal" | "vertical";
+  /**
+   * Direction of the slider
+   * @default false
+   */
+  reversed?: boolean;
+  /**
+   * True, if the direction of the slider is reversed
+   * @default false
+   */
+  isReversed?: boolean;
+  /**
+   * Get the formated value based on number format options
+   */
+  formatOptions?: Intl.NumberFormatOptions;
+  /**
+   * Get the value when it changes
+   */
+  onChange?: (values: number[]) => void;
+  /**
+   * Get the value when dragging is started
+   */
+  onChangeEnd?: (value: number[]) => void;
+  /**
+   * Get the value when dragging is stopped
+   */
+  onChangeStart?: (value: number[]) => void;
+}
+
+export const App: React.FC<AppProps> = args => {
+  const { label, isReversed, origin: originProp, onChange, ...rest } = args;
```

Comment:
Property 'origin' does not exist on type 'PropsWithChildren<AppProps>'

add the origin in AppProps

Reply:
Not needed !!